src/pages/frontpage/FrontPage.tsx

```diff
@@ -2,15 +2,25 @@ import { FrontPageHeading } from './FrontPageHeading';
 import { StyledPageContent } from '../../components/styled/Wrappers';
 import { SearchSection } from './SearchSection';
 import { NvaDescriptionSection } from './NvaDescriptionSection';
+import { Box } from '@mui/material';
```

Comment:
Om du har auto-sort i editor flyttes alle imports dit de hører hjemme rent alfabetisk. @joachimjorgensen  har dette for IntelliJ/web storm mener jeg?